docs_src/src/pages/documentation/api_reference/robyn_env.mdx

```diff
@@ -16,6 +16,9 @@ Batman wanted to configure the server through an environment file. Changing code
  - `ROBYN_BROWSER_OPEN`: Open the browser on successful start.
     - Default: `False`
     - Example: `ROBYN_BROWSER_OPEN=True`
+ - `ROBYN_DEV_MODE`: Configures the dev mode
+    - Default: `False`
```

Comment:
One follow up.

If we change it to `ROBYN_DEV_MODE=True` during execution, does it change to dev mode?